apps/consulting/pages/post/[slug].tsx

```diff
@@ -47,6 +48,11 @@ const Article: FC<TLibraryArticleProps> = ({
         description={post.meta.description}
         variant={DomainVariant.Quansight}
       />
+      <SocialCard
+        title={post.meta.title}
+        description={post.meta.description}
+        variant={DomainVariant.Quansight}
+      />
```

Comment:
> I'm guessing we'll have to do more work to make sure that the images are the right dimensions for Twitter and Open Graph

Exactly this, Twitter is 1:1, OpenGraph is 1.91:1. I doubt either exactly matches the ratio for the `featuredImage`; and, for some posts, we will want to be able to specify images of the correct ratio.

Reply:
I think both Storyblok and Vercel have image resizing/cropping features... we might be able to use them here... but will take some looking into

Reply:
Following up here; we're implementing so that we have the option for either the 1:1 `summary` Twitter card or the 1.9:1 `summary_large_image` card.

Thus, we'll have to also add metadata to indicate which type of Twitter card is desired.

Reply:
Created #738